include/kernel.h

```diff
@@ -1363,7 +1363,8 @@ static ALWAYS_INLINE s32_t _ms_to_ticks(s32_t ms)
 	/* use 64-bit math to keep precision */
 	return (s32_t)ceiling_fraction(
 		(s64_t)ms * sys_clock_hw_cycles_per_sec,
-		(s64_t)MSEC_PER_SEC * sys_clock_hw_cycles_per_tick);
+		((s64_t)MSEC_PER_SEC * sys_clock_hw_cycles_per_sec) /
+		 sys_clock_ticks_per_sec);
```

Comment:
I think I had an extra space prefixed here.

Reply:
you did, fixed now